db/migrate/20150128155644_allow_null_for_adviser_covers_whole_of_uk.rb

```diff
@@ -0,0 +1,5 @@
+class AllowNullForAdviserCoversWholeOfUk < ActiveRecord::Migration
+  def change
+    change_column :advisers, :covers_whole_of_uk, :boolean, null: true, default: nil
```

Comment:
About the after_initialize would be an idea, yeah. 
@lshepstone I just asked because I saw the commit message of 357a77613cd8588175d1ab43d24a5f4bcd724536 and thought: 'Ok, but why?'. And yes, sometimes i make that same questions with my commit messages, hehe. Someone in the future could make the same questions about it, ;).